stable/insights-agent/README.md

```diff
@@ -40,6 +40,25 @@ There are several different report types which can be enabled and configured:
 
 See below for configuration details.
 
+## Fleet Installation
+If you're installing the Insights Agent across a large fleet of clusters,
+it can be tedious to use the UI to create each cluster, then copy out the
+cluster's access token.
+
+To better serve customers with a large number of clusters, we've created a flow
+that allows you to easily deploy the Insights Agent across your fleet. You'll
+simply need to set the following flags:
```

Comment:
you probably need to add `insights.organization` is required here too.

Reply:
👍 good catch